vc-generator/index.js

```diff
@@ -1,8 +1,11 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ * Copyright 2023-24 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
```suggestion
 * Copyright 2023 - 2024 Digital Bazaar, Inc. All Rights Reserved
```

Use whole years for date ranges.